python/cuml/test/test_pickle.py

```diff
@@ -40,13 +40,18 @@
 decomposition_models = dict(
     PCA=cuml.PCA(),
     TruncatedSVD=cuml.TruncatedSVD(),
-    UMAP=cuml.UMAP()
+    UMAP=cuml.UMAP(),
+    GaussianRandomProjection=cuml.GaussianRandomProjection(),
+    SparseRandomProjection=cuml.SparseRandomProjection()
 )
 
 neighbor_models = dict(
     NearestNeighbors=cuml.NearestNeighbors()
 )
 
+dbscan_model = dict(

```

Comment:
shouldn't dbscan be in a 'cluster_models`?

Reply:
It's because of dbscan's API not having a `predict `method, unlike other cluster models